observability/logging.go

```diff
@@ -276,6 +248,45 @@ func (l *binaryLogger) Close() {
 	}
 }
 
+func createBinaryLoggerConfig(filters []*configpb.ObservabilityConfig_LogFilter) *iblog.LoggerConfig {
+	config := &iblog.LoggerConfig{
+		Services: make(map[string]*iblog.MethodLoggerConfig),
+		Methods:  make(map[string]*iblog.MethodLoggerConfig),
+	}
+	// Try matching the filters one by one, pick the first match. The
+	// correctness of the log filter pattern is ensured by config.go.
+	for _, filter := range filters {
+		if filter.Pattern == "*" {
+			// Match a "*"
+			if config.All != nil {
+				logger.Warningf("Ignored log_filter config: %+v", filter)
```

Comment:
As mentioned by Sanjay, it might be nice to only log a warning if the config is different from the older config.  (And below where we ignore the `svc/*` and `svc/method` patterns.)

Reply:
Done. Added.